app/lib/efile/ny/it214.rb

```diff
@@ -24,8 +23,9 @@ def calculate
           offboard
           return
         end
-        set_line(:IT214_LINE_4, -> { @claimed_as_dependent })
-        if @claimed_as_dependent
+        # Dependent on federal return?
```

Comment:
this field actually doesn't have to do with claimed_as_dependent and is actually an indicator for if a dependent is on the federal return, the 214 XML has a lot of errors which you can see when you force an inclusion in the supported docs. We probably want to make a ticket to go through and debug these